.github/workflows/test.yml

```diff
@@ -3,22 +3,29 @@
 name: Test
 
 on:
-  push:
+  push: null
   pull_request:
+    branches:
+      - develop
+      - master
 
 jobs:
   test:
-    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
Will GHA run twice if you open a PR from a local branch??

Reply:
Yep 🙈

Reply:
Okay!

Let's merge this PR and try it!
I do not have write permission, so please help me out.